prompts/add-info.txt

```diff
@@ -0,0 +1,77 @@
+
+You're part of a chain of agents whose final goal is to generate test files based on api specification snippets
+
+The tests will be part of a Typescript, Mocha, Chai and Axios testing framework that is capable of
+performing assertions on HTTP REST API calls.
+
+Test files use Service files and Models (interfaces) to make API calls. Service files encapsulate API calls in functions, one HTTP verp to one function.
+
+## Service Example Start ##
+
+import {{ ServiceBase }} from './ServiceBase.js'; // Import the base class
+
+export class BookingService extends ServiceBase {{
+  constructor() {{
+    super("/booking"); // Set the endpoint path
+  }}
+
+  async getBooking<T>(id: number, config = this.defaultConfig): Promise<Response<T>> {{
+    return await this.get<T>(`${{this.url}}/${{id}}`, config); // Use the inherited GET method
+  }}
+}}
+
+## Service Example End #
+
+Another part of the agent chain will be in charge of generating tests similar to the following example.
+
+## Test Example Start ##
+
+describe("Get Booking", () => {{
+  const bookingService = new BookingService();
+
+  it("@Smoke - Get Booking successfully - 200", async () => {{
+    const bookingId = 123456;
+    const response = await bookingService.getBooking<BookingModel>(bookingId);
+    response.status.should.equal(200, JSON.stringify(response.data));
+  }});
+  ...
+## Service Example End ##
+
+Naturally, some tests must use preconditions/setup/teardown logic that would require building before/beforeEach/after/afterEach blocks with additional API calls.
+The API calls in these preconditions can belong to the Service file matching the Test but they don't necessarily have to
+(eg: The Shipments .spec test file would use its corresponding Shipments service, but it could be necessary to use the Customers and Transactions service within the Shipments .spec test file)
+
+### TASK DESCRIPTION START ###
+
+Your task is to decide if its necessary to read additional files given a cetain verb_chunk and additional_context
```

Comment:
This should not be tied to the user's additional context.

Reply:
I thought of additional context and interconnected service dependencies as a single problem to tackle.
That is the reason why I decided to include the additional context in the prompt that tasks the model with gathering more info.
My main concern was to avoid having the model guess relations based on its own inherent knowledge, knowing this inherent understanding tends to 'generalize' and it is not necessarily aligned with our specific needs in the realm of connections between API sections.